aiohttp_devtools/runserver/config.py

```diff
@@ -123,7 +123,7 @@ def import_app_factory(self):
         :return: (attribute, Path object for directory of file)
         """
         rel_py_file = self.py_file.relative_to(self.python_path)
-        module_path = str(rel_py_file).replace('.py', '').replace('/', '.')
+        module_path = str(rel_py_file).replace('.py', '').replace('/', '.').replace('\\', '.')
```

Comment:
It might be a better idea to use different path class with just one fs path delimiter.

Reply:
I'd prefer working with Path object directly:
```python
path_parts = rel_py_file.parts
mod_path_parts[-1] = path_parts[-1].rstrip('.py')
module_path = '.'.join(mod_path_parts)
```

P.S. You may add this trailer to the commit:
```
Co-authored-by: Sviatoslav Sydorenko <wk@sydorenko.org.ua>
```

Reply:
Better:
```
mod_path_parts = rel_py_file.parts
module_path = '.'.join(mod_path_parts[:-1] + (mod_path_parts[-1].rstrip('.py'), ))
```